packages/botonic-plugin-contentful/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "scripts": {
     "build": "rm -rf lib && ../../node_modules/.bin/tsc",
+    "build:watch": "../../node_modules/.bin/tsc --watch",
```

Comment:
in order to not duplicate the same command for this, you can do:
`"build:watch": "npm run build -- --watch"` to pass variables through a npm script

Reply:
I thought about it too, but in this case, there's no need to remove the `lib` folder when executing the build with the `--watch` flag, that's why I was "duplicating" the command.

Reply:
as the watch process will be running in long term, I think that lib won't be removed every time (I haven't tested it, but I think it behaves like this). Not an important thing. Up to you!